bindings/experimental/distrdf/python/DistRDF/__init__.py

```diff
@@ -152,4 +156,7 @@ def create_distributed_module(parentmodule):
     # Set non-optimized default mode
     distributed.optimized = False
 
+    from .Profiling import ClingProfile
```

Comment:
This is another place where we are importing the Profiling module, thus activating the `CLING_PROFILE` environment variable irrespective of whether the user wants to profile or not. Unfortunately, this is breaking the CI. The tests run within the same Python session. The first distrdf test that is run will activate the environment variable. This leads to cling having debug symbols enabled, making the tests much slower. Finally the CI times out. We probably have to find some way to either hide this environment variable or activate the profiling feature programmatically